src/execution_plan/ops/op_update.c

```diff
@@ -143,6 +143,10 @@ void _UpdateSchemas(const OpUpdate *op) {
         LabelStoreType t = (ge->t == N_ENTITY) ? STORE_NODE : STORE_EDGE;
 
         LabelStore *store = LabelStore_Get(op->ctx, t, op->graphName, l);
+        // TODO if this label does not exist, we will not make any updates.
+        // Verify that this behavior matches the Cypher specification.
+        if (!store) return;
```

Comment:
To implement the alternate behavior, I think the logic at `src/execution_plan/ops/op_create.c:187` should be fine.

I'm not sure which is more correct, though!